subtests/docker_cli/systemd/p4321.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=docker-autotest systemd subtest. If you see this file on a live system it means something has failed very badly
+After=docker-latest.service
```

Comment:
It might be an overkill but I have the following method to edit the unit file depending on what docker_path is set.
   def edit_unit_file(self, unit_file):
        # edit systemd unit file and set it to value of docker_path
        filedata = None
        dockerpath = self.config['docker_path']
        docker_service = dockerpath.split('/')[-1] + ".service"
        with open(unit_file,'r') as file:
            for line in file:
                if 'After' in line:
                    docker_ufile = line.split('=')[-1].split('.')[0]
                    docker_ufile_path = "/usr/bin/" + docker_ufile
                    service_line = docker_ufile + ".service"
        with open(unit_file,'r') as file:
            filedata = file.read()
            if dockerpath != docker_ufile_path:
                filedata = filedata.replace(docker_ufile_path, dockerpath)
                filedata = filedata.replace(service_line, docker_service)
        with open(unit_file,'w') as file:
            file.write(filedata)